scripts/dev/update_nist_escalation.py

```diff
@@ -137,12 +137,12 @@ def parse_encost(fpath: Path) -> Tuple[List[dict], int]:
             continue
 
         if region is None:
-            region = line
+            region = line.expandtabs(1)
             # print(f"{region=}")
             continue
 
         if years is None:
-            years = [int(x) for x in line.split(" ")]
+            years = [int(x) for x in line.split()]
```

Comment:
same question really. What other types of whitespaces other than `" "` do you have? tabs again?

Reply:
Yes, tabs.

Reply:
The other changes were just because I was running on a Windows machine and had the files in a different place.